src/bin/publish_snapshot.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo "TRAVIS_BRANCH='$TRAVIS_BRANCH'"
+echo "TRAVIS_PULL_REQUEST='$TRAVIS_PULL_REQUEST'"
+
+if [ "$TRAVIS_BRANCH" == "develop" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+    sbt \
+        'set test in Test := {}' \
```

Comment:
Only invoked [`after_success`](https://github.com/broadinstitute/lenthall/pull/2#discussion-diff-41996488R8). No need to wait for them to run again.